src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Listing/Listing.html

```diff
@@ -34,7 +34,9 @@ <h3 class="sort-header">{{ "TR__SORT_BY" | translate }}</h3>
             </a>
         </div>
     </div>
-    <div class="listing-counter" data-ng-if="counter && elements.length > 0">{{elements.length}} {{contentType | adhResourceName | translate}}</div>
+    <div class="listing-counter" data-ng-if="counter && counterValue > 0">{{counterValue}} {{contentType | adhResourceName | translate}}</div>
+    <div class="listing-counter" data-ng-if="counter && noCounterValue && totalCount == 1">{{totalCount}} {{contentType | adhResourceName | translate}}</div>
+    <div class="listing-counter" data-ng-if="counter && noCounterValue && totalCount > 1">{{totalCount}} {{contentType | adhResourceName:true | translate}}</div>
```

Comment:
> In order to allow automatic pluralization, it would be better to change the adhResourceName filter to accept a number instead of a boolean. Then we could just pass in the count and the filter decides which variant to use.

I'll do that (but in `getNames`, not in `resourceName`).